MovieApp/MovieApp/App/Features/Movies/Components/TableViewCell/CategoryTableViewCell.swift

```diff
@@ -53,6 +53,13 @@ class CategoryTableViewCell: UITableViewCell {
     func set(delegate: HomeViewController) {
         self.delegate = delegate
     }
+    
+    @objc private func movieTapped(_ sender: UITapGestureRecognizer) {
+        guard let item = sender.view as? MovieCollectionViewCell else { return }
+        if let id = item.identifier {
+            delegate?.moviePressed(with: id)
+        }
+    }
```

Comment:
tap on a collectionView item should go through delegate of collectionView where you get an id of that viewModel and pass it to tableView

Reply:
Tried that before, but couldn't find a way to detect a touch on collection view cell. CollectionView has a built-in function _didSelectItemAt_ but it isn't usable because it requires selection and not a simple tap on cell. 

Therefore I used similar technique as one used when detecting which subcategory was pressed.